desk_sensor_table.py

```diff
@@ -0,0 +1,67 @@
+from config.config import CONFIG
+from desk import Desk
+from sensors.omron_d6t.omron_d6t import OmronD6T
+
+
+class DeskSensorTable:
+    """
+    This class is responsible for mapping desk objects to Sensors, and polling sensors occupancy data
+    """
+
+    # In this mapping, each entry is a tuple containing the desk id, the sensor object, and the respective desk object
+    # The purpose of desk id in the tuple is used for easy lookup of desks
+    # The sensor object is the object responsible for reading sensor data and reporting occupancy status
+    # The desk object is used for performing additional operations on the desk if necessary
+    __desk_mapping = []
```

Comment:
Could this be a dictionary with `id` as key? It would make `get_mapping_from_desk_id` a lot easier.